src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+
+But remember Goodhart's Law. When a metric becomes a target, it stops being a reliable metric.
+
+A perfect score doesn't guarantee full accessibility. You should treat Lighthouse accessibility scores as a starting point for evaluation rather than a final goal. Still, tracking these scores offers a valuable snapshot of the web's overall progress.
+
+Our report focuses exclusively on HTML and doesn't include PDF or other office documents.
+
+Compared to 2024, the median Lighthouse Accessibility score improved by 1%, reaching over 85% in 2025. Since the first Web Almanac in 2019, we've seen steady and incremental progress. <a hreflang="en" href="https://developer.chrome.com/docs/lighthouse/accessibility/scoring">Google Lighthouse assigns different weights</a> to axe-core issues, so <a hreflang="en" href="https://accessibility.civicactions.com/posts/prioritizing-accessibility-bugs-for-maximum-impact">organizations may prioritize fixes differently</a>.
+
+{{ figure_markup(
+  image="lighthouse-audit-improvements-yoy.png",
+  caption="Lighthouse audit improvements year-over-year.",
+  description="A bar chart showing the average increase in the media Google Lighthouse accessibility score over time for six years. Values increase slowly year by year, as follows: in 2019 83%, in 2020 80%, in 2021 82%, in 2022 83%, in 2024 84%, and in 2025 85%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQFD-7C6Jv6q1JyviDsKosRlVwaok7g7nRCQ9NGMw5MaAAohL7EcDejVwgp13Z_T2S_57Zi0YaVb7st/pubchart?oid=1116937150&format=interactive",
+  sheets_gid="1543303999",
+  sql_file="lighthouse_a11y_score.sql"
+  )
+}}
+
+This year, we've seen the biggest advances in the following axe-core tests:
+
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-input-field-name">ARIA input fields must have an accessible name</a>: +3% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-meter-name">ARIA `meter` nodes must have an accessible name</a>: +15% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-progressbar-name">ARIA `progressbar` nodes must have an accessible name</a>: +5% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-tooltip-name">ARIA `tooltip` nodes must have an accessible name</a>: +13% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/meta-refresh">Avoid delayed refresh under 20 hours</a>: +1% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/object-alt">`<object>` elements must have alternate text</a>: +1% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/select-name">`<select>` element must have an accessible name</a>: +5% over 2024
+
+{{ figure_markup(
+  image="most-improved-lighthouse-accessibility-tests-axe.png",
```

Comment:
Nit, I've reduced decimal places on this graph so X-axis reads a little neater. Please retake the PNG.